modules/bootstrapped/test/src/smithy4s/builder/ServiceBuilderSpec.scala

```diff
@@ -0,0 +1,72 @@
+package smithy4s
+
+
+import munit._
+import smithy4s.kinds.PolyFunction5
+import smithy4s.example.FooServiceGen
+
+class ServiceBuilderSpec extends FunSuite {
+
+  val service = smithy4s.example.FooService
```

Comment:
**nitpick:** I would've liked to see a service with multiple endpoints used in this test - allows for covering more ground in the tests (e.g. you see that endpoints aren't all replaced with the same thing)